.github/workflows/template_automerge_dependabot.yml

```diff
@@ -37,7 +37,10 @@ jobs:
           github-token: ${{ steps.get_token.outputs.token }}
 
       - name: Enable auto-merge for Dependabot PRs
-        if: steps.metadata.outputs.update-type == 'version-update:semver-patch' || steps.metadata.outputs.update-type == 'version-update:semver-minor'
+        if: >-
```

Comment:
Where did you found this nice pice?

Reply:
Somewhere in SO. Probably [here](https://stackoverflow.com/a/67532120) or [here](https://stackoverflow.com/a/65808412). 
You can read more on the difference between `|`, `>` and `>-` [here](https://stackoverflow.com/a/21699210) or in [official yaml specs](https://yaml.org/spec/1.2-old/spec.html#id2760844).